docs/docs/reference/metaprogramming/tasty-inspect.md

```diff
@@ -33,4 +36,18 @@ object Test {
     ConsumeTasty("", List("foo.Bar"), new Consumer)
   }
 }
-```
\ No newline at end of file
+```
+
+Note that if we need to run the main (in an object called `Test`) after
+compilation we need make available the compiler to the runtime:
+
+```shell
+dotc -with-compiler -d out Test.scala
+dotr -with-compiler -classpath out Test
+```
+
+Or, from SBT:
+
+```scala
+libraryDependencies += "ch.epfl.lamp" %% "dotty-tasty-consumer" % scalaVersion.value
```

Comment:
This is the most important information and should be at the top, not at the bottom.